cmake/stm32/linker_ld.cmake

```diff
@@ -16,6 +16,10 @@ _eccmram = .;\n\
 } >CCMRAM AT> FLASH\n\
         ")
 endif()
+
+if(NOT USER_DATA_FLASH_SIZE)
+    set(USER_DATA_FLASH_SIZE "0K")
```

Comment:
Why "OK" and not 0 since you use this in a substraction afterwrards?

Reply:
It's 0K, not OK and it's because the .ld expects a unit after the number